.github/CONTRIBUTING.Rmd

```diff
@@ -0,0 +1,185 @@
+
+| Label name         | Description                                                                                                                        |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `SPx`              | `SP` (story points) indicate complexity, and the larger the subsequent number, the more time consuming the issue is expected to be |
+| `priority`         | Issues with this label should be completed with higher priority                                                                    |
+| `good first issue` | Good choices for new team members                                                                                                  |
+| `blocked`          | Blocked by other issues                                                                                                            |
+| `bug`              | Something isn't working                                                                                                            |
+| `devops`           | Development and Operation                                                                                                          |
+| `discussion`       | Disccusion needed                                                                                                                  |
+| `documentation`    | Improvement of documentation is needed                                                                                             |
+| `duplicate`        | The issue already exists                                                                                                           |
+| `enhancement`      | New feature or request                                                                                                             |
+| `help wanted`      | Extra attention is needed                                                                                                          |
+| `invalid`          | This doesn't seem right                                                                                                            |
+| `question`         | Further information needed                                                                                                         |
+
+Please choose an issue based on your interest, issue complexity, and priority.
+
+### Add new unit tests
+
+To add a new unit test, you need to first identify the test scope.
+Does the test fit in the scope of existing tests?
+If so, please modify the existing test files under `tests/testthat/` folder or `src/` folder, depending on whether the code to be tested is R or C++.
+Otherwise please create a new test file, with a name prefix of "test-".
+
+In each test case, use the following structure:
+
+```{r eval=FALSE}
+test_that("<function_name> do something as explained", {
+  # test body
+  expect_identical(1, 1)
```

Comment:
funnily enough, I've provided boilerplate like this to contributors and eventually found `expect_identical(1,1)` in their eventual first PR.

Reply:
thanks, I made it even more obvious now :-)